src/steam_network/user_info_cache.py

```diff
@@ -31,29 +31,31 @@ def is_initialized(self) -> bool:
 
 
     def to_dict(self):
-        creds = {
-            'steam_id': base64.b64encode(str(self._steam_id).encode('utf-8')).decode('utf-8'),
-            'refresh_token': base64.b64encode(str(self._refresh_token).encode('utf-8')).decode('utf-8'),
-            'account_username': base64.b64encode(str(self._account_username).encode('utf-8')).decode('utf-8'),
-            'persona_name': base64.b64encode(str(self._persona_name).encode('utf-8')).decode('utf-8'),
-        }
+        creds = {}
+        if self.is_initialized():
+            creds = {
+                'steam_id': base64.b64encode(str(self._steam_id).encode('utf-8')).decode('utf-8'),
+                'refresh_token': base64.b64encode(str(self._refresh_token).encode('utf-8')).decode('utf-8'),
+                'account_username': base64.b64encode(str(self._account_username).encode('utf-8')).decode('utf-8'),
+                'persona_name': base64.b64encode(str(self._persona_name).encode('utf-8')).decode('utf-8'),
+            }
         return creds
 
     def from_dict(self, lookup):
         for key in lookup.keys():
             if lookup[key]:
                 logger.info(f"Loaded {key} from stored credentials")
```

Comment:
```suggestion
        for key, val in lookup.items():
            if val:
                logger.info(f"Loaded {key} from stored credentials")
```

another style suggestion feel free to ignore